src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="big-box">
+      <label for="search-big">
+        <input
+          type="text"
+          name="search-big"
+          id="search-big"
+          data-qa="keypress"
+          placeholder="Try &#34;Los Angeles&#34;"
+          class="search-box search-box-big"
+        >
+        <img
+          src="./images/Search.svg"
+          alt="Icon for search input"
+          class="icon-search-big"
+        >
+      </label>
+    </form>
+    <form data-qa="small" class="small-box">
+      <label for="search-small">
+        <input
+          type="text"
+          name="search-small"
+          id="search-small"
+          placeholder="Try &#34;Los Angeles&#34;"
+          class="search-box search-box-small"
+        >
+        <div class="icon-search-small">
```

Comment:
why here is div, but un the bigger one there are any divs? let us remove this one